src/main/java/org/threadly/litesockets/tcp/ssl/SSLClient.java

```diff
@@ -246,37 +254,32 @@ public boolean isEncrypted() {
       if(ssle.getHandshakeStatus() == NEED_WRAP) {
         writeForce(ByteBuffer.allocate(0));
       }
-      if(this.ce != null) {
-        ce.getThreadScheduler().schedule(new Runnable() {
+      if(this.seb != null) {
```

Comment:
What problem did you discover?

Why do you need a "TimeoutException"?  If a .get() ended up throwing an unchecked CancellationException, would that not work?  (just asking because I want to figure out how to make Watchdog awesome)